go.mod

```diff
@@ -258,3 +258,5 @@ replace github.com/MariusVanDerWijden/tx-fuzz => github.com/marcopolo/tx-fuzz v0
 
 // See https://github.com/prysmaticlabs/grpc-gateway/issues/2
 replace github.com/grpc-ecosystem/grpc-gateway/v2 => github.com/prysmaticlabs/grpc-gateway/v2 v2.3.1-0.20220721162526-0d1c40b5f064
+
+replace github.com/ethereum/go-ethereum => github.com/ethereum/go-ethereum v1.10.24-0.20221209152046-3775e198dfc9 // required until commit 90711efb0ab6360c37ab2135ab961daf05e83a74 is included in a tagged release
```

Comment:
Can't we add go-ethereum like this if we need https://github.com/ethereum/go-ethereum/commit/90711efb0ab6360c37ab2135ab961daf05e83a74?
```
go get github.com/ethereum/go-ethereum@90711efb0ab6360c37ab2135ab961daf05e83a74
```

Reply:
Disclaimer: I am a go mod n00b

When I tried to go-get that specific commit, a bunch of dependencies get downgraded by go mod tidy and then it complains about ambiguous imports.

if I try to go-get a more recent geth commit, then I get:

```
[prysm]: go get github.com/ethereum/go-ethereum@3775e198dfc9fec312f44fd563dcc525cefa1e5d
go: github.com/ethereum/go-ethereum@3775e198dfc9fec312f44fd563dcc525cefa1e5d (v1.10.24-0.20221209152046-3775e198dfc9) requires github.com/ethereum/go-ethereum@v1.10.26, not github.com/ethereum/go-ethereum@3775e198dfc9fec312f44fd563dcc525cefa1e5d (v1.10.24-0.20221209152046-3775e198dfc9)
```